bindings/java/wasmedge-java/src/main/java/org/wasmedge/WasmEdge.java

```diff
@@ -15,7 +15,11 @@ public class WasmEdge {
     private static final String NATIVE_LIBRARY_NAME = "wasmedge_jni";
 
     public static synchronized void init() {
-        System.loadLibrary(NATIVE_LIBRARY_NAME);
+        try {
+            NativeUtils.loadLibraryFromJar("/" + System.mapLibraryName(NATIVE_LIBRARY_NAME));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
If the code is only rewrap the exception, why not let `loadLibraryFromJar` throws correct type?

Reply:
Also, it's danger to use `"/" + System...`. Do we have any reason to avoid `loadLibraryFromJar` handle these?

Reply:
Changed `loadLibraryFromJar` to throw relevant `RuntimeException`s. The reasoning is that I didn't want to introduce breaking changes to the signature of `WasmEdge#init`.

Also, it's not `"/"` meant in the OS `"/"` way. In this case, it looks for the dynamic lib at root of the jar archive (`"/"`) and copies it to `/tmp/wasmedge_jni_tmp/` in the actual filesystem.